app/views/canvas/full_zoom/_my_module.html.erb

```diff
@@ -1,4 +1,7 @@
-<div class="panel panel-default module-large<%= " alert-red" if my_module.is_overdue? %><%= " alert-yellow" if my_module.is_one_day_prior? %>"
+<div class="panel panel-default module-large
+  <%= " alert-red" if my_module.is_overdue? && !my_module.completed? %>
```

Comment:
You can dry out this and create ta method in canvas_helper that handle this logic